org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_Positioner.java

```diff
@@ -93,20 +93,20 @@ public IPosition getPosition() throws ScanningException {
 	}
 
 	@Override
-	public List<IScannable<?>> getMonitors() throws ScanningException {
+	public List<IScannable<?>> getMonitorsPerPoint() throws ScanningException {
 		// TODO Use the _Scannable which is a remote scannable connection.
 		return null;
 	}
 
 	@Override
-	public void setMonitors(List<IScannable<?>> monitors) {
+	public void setMonitorsPerPoint(List<IScannable<?>> monitors) {
 		// TODO Use the _Scannable which is a remote scannable connection.

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)